.github/workflows/main.yml

```diff
@@ -0,0 +1,20 @@
+name: CI
+
+on: 
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: WordPress Plugin Deploy
+      uses: 10up/action-wordpress-plugin-deploy@1.4.0
+      env:
+        SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
```

Comment:
where is this stored?

Reply:
https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets

They're encrypted variables saved on in the repo's settings, even I can't view the values after setting them.